src/Tools/ExternalAccess/LiveShare/StringConstants.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.LiveShare
+{
+    internal class StringConstants
+    {
+        public const string BaseRemoteAssemblyTitle = "Base Remote Language Service";
+
+        // The service name for an LSP server implemented using Roslyn designed to be used with the Roslyn client
+        public const string RoslynContractName = "Roslyn";
+        // The service name for an LSP server implemented using Roslyn designed to be used with the LSP SDK client
+        public const string RoslynLSPSDKContractName = "RoslynLSPSDK";
```

Comment:
>RoslynLSPSDKContractName [](start = 28, length = 24)

Style: Naming guidelines for 3+ letter abbreviations are to capitalize first letter and make the rest lower case. So, this should be `RoslynLspSdkContractName`